docs/reference/ingest/processors/community-id.asciidoc

```diff
@@ -0,0 +1,91 @@
+[role="xpack"]
+[testenv="basic"]
+[[community-id-processor]]
+=== Community ID processor
+++++
+<titleabbrev>Community ID</titleabbrev>
+++++
+
+Computes the Community ID for flow data as defined in the
+https://github.com/corelight/community-id-spec[Community ID Specification].
+
+[[community-id-options]]
+.Community ID Options
+[options="header"]
+|======
+| Name               | Required | Default       | Description
+| `source_ip`        | no       | `source.ip`   | Field containing the source IP address.
+| `source_port`      | no       | `source.port` | Field containing the source port.
+| `destination_ip`   | no       | `destination.ip` | Field containing the destination IP address.
+| `destination_port` | no       | `destination.port` | Field containing the destination port.
+| `iana_number`      | no       | `network.iana_number` | Field containing the IANA number.
+| `icmp_type`        | no       | `icmp.type`   | Field containing the ICMP type.
+| `icmp_code`        | no       | `icmp.code`   | Field containing the ICMP code.
+| `transport`        | no       | `network.transport` | Field containing the transport protocol.
```

Comment:
Should we mention that this field is only used when the iana_number field does not exist?